lib/datatable/InfineonDatatable.vue

```diff
@@ -32,11 +32,26 @@
               class="text-nowrap"
               scope="col"
             >
-              {{ column.title }}
+              <!--default column title slot - hidden for slot with specified column title-->
+              <slot
+                v-if="!$slots[`col.${column.title}`]"
+                name="col"
+                v-bind="column"
+              >
+                {{ column.title }}
+              </slot>
+
+              <!--special column title slot-->
+              <slot
+                :name="`col.${column.title}`"
+                v-bind="column"
+              />
+
```

Comment:
ah wahnsinn, dass das so funktioniert :D

              <slot  :name="`col.${column.title}`"
                v-bind="column"
              >  {{ column.title }} </slot>

sollte eigentlich funktionieren anstatt Zeile 35-49.

column.title is "fallback content" wenn kein slot angegeben ist.
Siehe doku hier:
https://vuejs.org/guide/components/slots.html#fallback-content

Reply:
wenn ich hier den column title nochmal angebe, wird er allerdings doppelt angezeigt für die default columns